react.js

```diff
@@ -18,24 +26,29 @@ for(let i=0; i<names.length; i++){
 }
 
 
+console.log(names.map(x => (`${x} is ${x.length} characters long.`)))
 
-// 3. Destructure the following variables out of state.
+
+3. Destructure the following variables out of state.
 
 this.state = {
   name: "Dracula",
   home: "Transylvania",
   dislikes: ["mirrors", "garlic", "wooden stakes"]
 }
 
-
+const { name, home, dislikes } = this.state
 
 // 4. Write a React method that would add one and update the state of the count each time the method is called.
 
 this.state = {
   count: 0
 }
 
-
+addOne = (num) => {
+  this.state.count + 1
+  this.setState({count: newCount})
+}
 
 // 5. There are four mistakes in this code that would cause it to break our application. Find the mistakes and fix them:
 
```

Comment:
1) class Recipes extends Component
2) recipes should be an array so we can use the map function
3) the mapping logic should be outside the return
4) destructuring recipes out of state